docs/en/latest/nacos.md

```diff
@@ -108,3 +109,45 @@ Send a request to confirm whether service discovery is in effect:
 ```bash
 curl http://127.0.0.1:9080/get -H 'Host: httpbin'
 ```
+
+# Features
+
+## Metadata
```

Comment:
I suggest you give a complete best practice directly and include some necessary clarifications